test/index.test.js

```diff
@@ -2,19 +2,45 @@ const request = require("supertest");
 const app = require('../app.js');
 const testPerson = require('./testPerson.json')
 
-test("Test the get method", async (done) => {
-    const response = await request(app).get("/1");
-    expect(response.status).toBe(200);
-    expect(response.body.name).toBe('Luke Skywalker');
-    expect(response.body.films.length).toBe(5);
-    expect(response.body.vehicles.length).toBe(2);
-    expect(response.body.starships.length).toBe(2);
-    expect(response.body).toEqual(testPerson);
-    done();
-})
+jest.mock('got');
+
+describe('Positive tests',()=>{
+    let response;
+
+    beforeEach(async () => {
+        response=await request(app).get("/1");
+    });
+    
+    test("response status is okay", async (done) => {
+        expect(response.status).toBe(200);
+        done();
+    });
+
+    test("name of the person fetrched is same",async (done)=>{
```

Comment:
I need you to look into using code formatting tools like "prettier" for example. Your consistency in code formatting is all over the place.